pkg/apis/servicecatalog/serialization_test.go

```diff
@@ -156,7 +157,7 @@ func roundTrip(t *testing.T, codec runtime.Codec, item runtime.Object) {
 		return
 	}
 
-	if !api.Semantic.DeepEqual(original, item) {
+	if !equality.Semantic.DeepEqual(original, item) {
```

Comment:
I believe the package change here is due to the new version of demantic dep equal that isn't broken by protobuf changes, right?